### PR TITLE
feat(userbutton): image size linked to icon size + default icon in img placeholder

### PR DESCRIPTION
**Pull request by ThibaultNocchi**
Putting the icon in the v-img placeholder makes it appear while the user image is loading (or not at
all) instead of the plain blue circle

Also idk about you, but I got this blue halo around the pic due to the background, I'll try to look into it
print_screen_2020-12-14-10-11-02

### CODE REVIEW

File: components/UserButton.vue [hunk 1]
```diff
@@ -2,13 +2,12 @@
...
-          <v-img
-            v-if="userImage"
-            :src="userImage"
-            :alt="$auth.user.Name"
-          ></v-img>
-          <v-icon v-else dark>mdi-account</v-icon>
+        <v-avatar :size="avatarSize" color="primary" class="mr-4">
+          <v-img :src="userImage" :alt="$auth.user.Name">
+            <template #placeholder>
+              <v-icon dark>mdi-account</v-icon>
```
Comment (Camc314) [inline components/UserButton.vue:8]: Can you confirm that changing from between light/dark theme keeps the icon the same color (white)?

Comment (ThibaultNocchi) [reply components/UserButton.vue:8]: Yup, the `dark` prop is static and not linked to the `$vuetify.theme.dark` property I've noticed. Here are the pics:
print_screen_2020-12-14-12-02-46
print_screen_2020-12-14-12-02-41

Comment (Camc314) [reply components/UserButton.vue:8]: Cool Thanks for checking

**Review by Camc314 — APPROVED** [review]

File: components/UserButton.vue [hunk 2]
```diff
@@ -51,7 +51,10 @@ export default Vue.extend({
     userImage: {
       get() {
```
Comment (MrTimscampi) [inline components/UserButton.vue:52]:
```suggestion
    userImage(): string {
```

This should help with the TypeScript error.

Comment (ThibaultNocchi) [reply components/UserButton.vue]: You're right, I didn't even think of it haha. Thanks!

**Pull request merged**